src/util/time.cpp

```diff
@@ -129,6 +120,8 @@ int64_t GetTimeSeconds()
     return int64_t{GetSystemTime<std::chrono::seconds>().count()};
 }
 
+int64_t GetTime() { return GetTime<std::chrono::seconds>().count(); }
```

Comment:
Nit: Why not using the same syntax/formatting as above in `GetTimeSeconds`?

Reply:
I think the narrowing check is a bit overkill and probably doesn't make much sense. Mostly relevant isn't the *maximum* number of bits in the value, but the *minimum*, which is 35 bits according to https://en.cppreference.com/w/cpp/chrono/duration .

In either case the function is deprecated (see comment in the header file), and will likely be removed in the future anyway.